### DISCUSSION TITLE
[Art competition] Abstract interpretaion

### DISCUSSION THREAD

**Discussion by yeonhee-ryou** [prosyslab-classroom/cs524-program-analysis#270] [Art competition]
![소믈리에](https://user-images.githubusercontent.com/17640199/203905663-6badba15-0bb9-4204-acf5-bb0fe4077020.png)

## Short Description
For me, [Abstract interpretaion] means [a painting of the sommelier who is imagining a wonderful and fantastic landscape]

## Long Description (up to 3 sentences)
Sommeliers can taste wine and feel not only the scent of grapes but also the scent of barrels. They can guess and evaluate from grape varieties to the environment where vine trees grew. Abstract interpretation is a delicate process like tasting wine, which examines abstract domains and semantics carefully to evaluate the safety properties of a program.

**Comment by KihongHeo** [comment]
What happens if this wine has a severe bug? (기미상궁?)

**Comment by yeonhee-ryou** [comment]
Then the wine won't be availabe at department stores 😞 Cheer up tasters and makers

**Comment by KihongHeo** [comment]
Good. Unfortunately, buggy programs are available at App Store.

**Comment by f52985** [comment]
Well, it seems that the real world sommelier are too healthy (or too imprecise) to detect the poison I guess :(
